src/utils/lang/index.js

```diff
@@ -157,6 +164,18 @@ export function uniq(arr) {
   });
 }
 
+/**
+ * Removes duplicate items on an array of objects using an optional `stringify` function as equality criteria.
+ * It uses JSON.stringify as default criteria.
+ */
+export function uniqO(arr, stringify = JSON.stringify) {
```

Comment:
This name is not representative at all. either call it "uniqAsStrings" or just preprocess the items before passing them to the regular uniq.

Reply:
Renamed. By passing the preprocessed items to regular uniq, we lose the reference of what original items were removed.